spec/10-expressions.md

```diff
@@ -616,6 +616,7 @@ object-creation-expression:
   'new' class-type-designator '(' argument-expression-list ','? ')'
   'new' class-type-designator
   'new' 'class' '(' argument-expression-list? ')' class-base-clause? class-interface-clause? '{' class-member-declarations? '}'
+  'new' 'class' '(' argument-expression-list ','? ')' class-base-clause? class-interface-clause? '{' class-member-declarations? '}'
```

Comment:
I believe this should either use `','` without `?`, or the existing line should be extended with `','?`. Otherwise, the two productions are ambiguous, code without the comma can be parsed by either. I see this is already wildly inconsistent in the grammar, so maybe I'm missing something.

Reply:
Maybe? I was trying to follow the patterns I saw elsewhere in the grammar, but I can't find any information on the specification for the grammar we use, so it's difficult to understand what is meant or intended in places.